infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -216,6 +216,7 @@ public static void main(String[] args) {
             opts.put("--bind-to-all-interfaces", false);
         }
 
+        log.info("Server arguments: {}", opts);
```

Comment:
Do you want to remove the debug statement above?

Reply:
Didn't actually see that log statement. Moved it just before the server starts listening on the port because some of the opts are changed after they are parsed.